pom.xml

```diff
@@ -174,7 +174,7 @@
         <dep.iceberg.version>1.3.0</dep.iceberg.version>
         <dep.protobuf.version>3.22.2</dep.protobuf.version>
         <dep.wire.version>4.5.0</dep.wire.version>
-        <dep.netty.version>4.1.92.Final</dep.netty.version>
+        <dep.netty.version>4.1.94.Final</dep.netty.version>
```

Comment:
Do you need 94? It's not working with apache arrow 12.0.1

Reply:
We need to wait for the release than contains https://github.com/apache/arrow/issues/36209

Reply:
We are now at   <version>4.1.101.Final</version>